index.html

```diff
@@ -166,5 +166,30 @@ <h3 class="skills"><img src="assets/Rectangle-3.png" alt="">Skills</h3>
             </section>
         </div>
     </div>
+    <section class="contact">
+        <div>
+            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form class="contact-form" method="post">
+                <input type="text" placeholder="Full name" id="name" name="user_name">
```

Comment:
- [ ] Kindly add `maxlength="30"` attribute`, to limit the number of characters entered as full names.